modules/programs/rofi.nix

```diff
@@ -266,9 +266,9 @@ in {
       rofiWithPlugins = cfg.package.override
         (old: rec { plugins = (old.plugins or [ ]) ++ cfg.plugins; });
     in if builtins.hasAttr "override" cfg.package then
-      rofiWithPlugins
+      cfg.package
     else
-      cfg.package;
+      rofiWithPlugins;
```

Comment:
I think the order was correct before? We can only use `cfg.package.override` if `cfg.package` has an `override` attribute...